core/src/Cabana_Parallel.hpp

```diff
@@ -249,6 +253,8 @@ inline void neighbor_parallel_for(
     const FunctorType& functor, const NeighborListType& list,
     const FirstNeighborsTag, const SerialOpTag, const std::string& str = "" )
 {
+    Kokkos::Profiling::pushRegion( "Cabana::neighbor_parallel_for" );
```

Comment:
Are these `neighbor_parallel_for ` regions really useful?  These seem to only a single kernel launch with nothing other than construct the policy.  Did you look at some profiles that made you think that we should add them?
A possible alternative would be to give a default value not empty to the `str` argument.  Presumably when provided by the user that label is good enough and we don't really need the region.

Reply:
That's a fair question. It is more useful for the Cabana algorithms/communication, but I do like showing where Cabana is being used (even when it effectively duplicates the user kernel in the profile)